### PR TITLE
Release

### PR DESCRIPTION
**Pull request by NSeydoux**
This PR bumps the version to 0.1.1.

# Checklist

- [X] I used `npm version <major|minor|patch>` to update `package.json`, inspecting the changelog to determine if the release was major, minor or patch.
- [X] The CHANGELOG has been updated to show version and release date - https://keepachangelog.com/en/1.0.0/.
- [X] The **only** commits in this PR are:
  - the CHANGELOG update.
  - the version update.
  - `@since` annotations.
- [X] I will make sure **not** to squash these commits, but **rebase** instead.
- [ ] Once this PR is merged, I will push the tag created by `npm version ...` (e.g. `git push origin vX.Y.Z`).

### CODE REVIEW

File: CHANGELOG.md
Comment (Vinnl) [inline CHANGELOG.md:11]: Actually I think the other year is incorrect too :)

```suggestion
## 0.1.1 - 2021-03-12
```

**Review by Vinnl — APPROVED** [review]

**Review by matthieubosquet — CHANGES_REQUESTED** [review]
Once the 2020 year is gone, it all looks good!

**Comment by Vinnl** [comment]
@matthieubosquet Small suggestion: if you select "Request changes", you'll have to do another review before this can be merged. Given that Nicolas is probably willing and able to fix the date such that another review probably wouldn't surface additional issues, I generally find that there's no need for that overhead, so I just approve it with suggestions and trust Nicolas to do the right thing :)

**Review by matthieubosquet — APPROVED** [review]
You're so right! I actually didn't know ^^,

Thanks @Vinnl ;)

File: CHANGELOG.md
Comment (NSeydoux) [reply CHANGELOG.md:11]: It's becoming a running gag to have wrong release dates ^^

Comment (Vinnl) [reply CHANGELOG.md]: Let's just pretend that it's bait to attract helpful friendly contributors :P

**Pull request merged**
